packages/cli/src/commands/model/list.ts

```diff
@@ -29,12 +28,19 @@ export default class ModelList extends Command<ModelListFlags> {
     }),
   }
 
+  getPageSize(): number {
+    const rows = terminalSize()['rows']
+    // When we display a table, each row is takes two terminal rows. We subtract additional 3 lines per page, to leave
+    // space for the table header and next page prompt
+    return this.flags.table ? rows / 2 - 3 : rows
```

Comment:
Shouldn't we ensure the output is a positive integer here? Seems like it could be a float and even negative if `rows` is lower than 6.